documentation/writing_tasks.md

```diff
@@ -967,6 +967,67 @@ Successful on 1 target: localhost
 Ran on 1 target in 0.12 seconds
 ```
 
+### PowerShell example
+
+Create task and metadata in your project at
+`<PROJECT>/tasks/mytask.{json,ps1}`.
```

Comment:
```suggestion
`<PROJECT DIRECTORY>/tasks/mytask.{json,ps1}`.
```
Asked about this on one of Tom's PRs too - we standardized on this a few months ago, but I'm not sure if `PROJECT` is better....